From 2dec614e8f2e4b21c4ea7b7c18a0c3cb20fca631 Mon Sep 17 00:00:00 2001
From: Alexander Forsch <alexander.forsch@bytesatwork.ch>
Date: Tue, 22 Jul 2025 14:38:26 +0200
Subject: [PATCH] arm64: freescale: imx93-bytedevkit.dts: fix usb

Fix support for usb-storage on byteDEVKIT i.MX93.
Note, only rev2 of byteENGINE i.MX93 is supported now.

Signed-off-by: Alexander Forsch <alexander.forsch@bytesatwork.ch>
---
 arch/arm64/boot/dts/freescale/imx93-bytedevkit.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx93-bytedevkit.dts b/arch/arm64/boot/dts/freescale/imx93-bytedevkit.dts
index 0a6a1fccd9..8ed3baf329 100644
--- a/arch/arm64/boot/dts/freescale/imx93-bytedevkit.dts
+++ b/arch/arm64/boot/dts/freescale/imx93-bytedevkit.dts
@@ -40,7 +40,7 @@ reg_usb_otg1_vbus: regulator-otg {
 		regulator-name = "USB1_VBUS";
 		regulator-min-microvolt = <5000000>;
 		regulator-max-microvolt = <5000000>;
-		gpio = <&gpio2 18 GPIO_ACTIVE_HIGH>;
+		gpio = <&gpio2 11 GPIO_ACTIVE_HIGH>;
 		enable-active-high;
 	};
 
@@ -49,7 +49,7 @@ reg_usb2_vbus: regulator-vbus {
 		regulator-name = "USB2_VBUS";
 		regulator-min-microvolt = <5000000>;
 		regulator-max-microvolt = <5000000>;
-		gpio = <&gpio2 19 GPIO_ACTIVE_HIGH>;
+		gpio = <&gpio2 18 GPIO_ACTIVE_HIGH>;
 		enable-active-high;
 		regulator-always-on;
 	};
@@ -469,13 +469,13 @@ MX93_PAD_SD2_VSELECT__GPIO3_IO19	0x51e
 
 	pinctrl_usb1: usb1_grp {
 		fsl,pins = <
-			MX93_PAD_GPIO_IO18__GPIO2_IO18		0x31e
+			MX93_PAD_GPIO_IO11__GPIO2_IO11		0x31e
 		>;
 	};
 
 	pinctrl_usb2: usb2_grp {
 		fsl,pins = <
-			MX93_PAD_GPIO_IO19__GPIO2_IO19		0x31e
+			MX93_PAD_GPIO_IO18__GPIO2_IO18		0x31e
 		>;
 	};
 
